homeassistant/components/pvpc_hourly_pricing/sensor.py

```diff
@@ -33,14 +33,29 @@ async def async_setup_entry(
     name = config_entry.data[CONF_NAME]
     pvpc_data_handler = PVPCData(
         tariff=config_entry.data[ATTR_TARIFF],
+        power=config_entry.data[ATTR_POWER],
+        power_valley=config_entry.data[ATTR_POWER_P3],
         local_timezone=hass.config.time_zone,
         websession=async_get_clientsession(hass),
         logger=_LOGGER,
```

Comment:
Side note: The library should not use the Home Assistant logger. The library should use its own logger(s).

Reply:
Ok, noted!

I did that a long time ago and I don't remember why 😅. It's an optional parameter, same as the websession, removing it